gno.land/Makefile

```diff
@@ -17,6 +17,10 @@ CGO_ENABLED ?= 0
 export CGO_ENABLED
 # test suite flags.
 GOTEST_FLAGS ?= -v -p 1 -timeout=30m
+# user for gnokey version [branch].[N]+[hash]
+VERSION ?= $(shell git describe --tags --exact-match 2>/dev/null || echo "$(shell git rev-parse --abbrev-ref HEAD).$(shell git rev-list --count HEAD)+$(shell git rev-parse --short HEAD)")
+
+GOBUILD_FLAGS ?= -ldflags "-X github.com/gnolang/gno/tm2/pkg/version.Version=$(VERSION)"
```

Comment:
What do you think about doing just:

```
VERSION ?= $(shell git describe --tags --long --dirty --always)

GOBUILD_FLAGS ?= -ldflags=-X=github.com/gnolang/gno/tm2/pkg/version.Version=$(VERSION)
```

I'm not entirely convinced we need the branch name, if the commit hash is there 👀 

cc @gfanton

Reply:
Also, don't we already version something like gnodev?